client/app/dbaas/logs/detail/options/logs-options.less

```diff
@@ -0,0 +1,43 @@
+.log-options {    
```

Comment:
We shouldn't have to use a lot of custom less... pretty much everything should be handled by the ui-kit or bootstrap utilities.

Reply:
@Mboulianne Yes, however, the mockups are more up-to-date than the components. So I'm expecting to see some variants. Depending on Étienne's review, we'll update the UI-Kit.

Reply:
@Mboulianne , I have removed a couple of css properties/classes by using bootstrap classes. But like @Netnix mentioned, the mocks require a few components in the UI-Kit to appear a bit different. Hence, I used css to style the elements inside the components used from UI-Kit.